libmachine/provision/configure_swarm.go

```diff
@@ -50,6 +50,9 @@ func configureSwarm(p Provisioner, swarmOptions swarm.Options, authOptions auth.
 			swarmOptions.Strategy,
 			advertiseMasterInfo,
 		)
+		if swarmOptions.IsExperimental {
```

Comment:
According to Swarm's [documentation](https://github.com/docker/swarm/tree/master/experimental), the accepted flag is with a single dash, that's why I'm using single dash.